setup.py

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Is that bit really needed? It looks like setup.py is not using unicode literals directly.